src/features/user/user.module.ts

```diff
@@ -2,9 +2,10 @@ import { Module } from '@nestjs/common';
 import { UserService } from './user.service';
 import { UserController } from './user.controller';
 import { UserJsonRepository } from './user.json-repository';
+import { AuthController } from '../auth/auth.controller';
 
 @Module({
-  controllers: [UserController],
+  controllers: [UserController, AuthController],
```

Comment:
Why UserModule respond for AuthController

Reply:
he is not needed here, deleted